<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Hanvika" failures="1" tests="5" name="Test" time="0.069" errors="0" timestamp="2025-07-29T21:16:03 IST">
  <testcase classname="testng.frameworkconcepts.Appmethods" name="testcase1" time="0.006">
    <failure type="java.lang.AssertionError" message="testcase1 failed for somereason">
      <![CDATA[java.lang.AssertionError: testcase1 failed for somereason
at testng.frameworkconcepts.Appmethods.testcase1(Appmethods.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
  <testcase classname="testng.frameworkconcepts.Appmethods" name="testcase2" time="0.001"/>
  <testcase classname="testng.frameworkconcepts.Appmethods" name="testcase3" time="0.007"/>
  <testcase classname="testng.frameworkconcepts.Appmethods" name="testcase3" time="0.001"/>
  <testcase classname="testng.frameworkconcepts.Appmethods" name="testcase3" time="0.001"/>
  <testcase classname="testng.frameworkconcepts.Webmethods" name="@AfterSuite stopreporting" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite stopreporting -->
  <testcase classname="testng.frameworkconcepts.Webmethods" name="@BeforeSuite startreporting" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite startreporting -->
</testsuite> <!-- Test -->
